code-experiments/src/coco.h

```diff
@@ -465,7 +465,7 @@ void coco_free_memory(void *data);
 /**
  * @brief Signals a fatal error.
  */
-void coco_error(const char *message, ...);
+void coco_error(const char *message, ...) __attribute__((noreturn)) __attribute__((__noreturn__));
```

Comment:
This seems to be the culprit for the AppVeyor failure

Reply:
I've hidden the attributes behind macros which are defined based on the used compiler. Let's hope this fixes it.